src/agent/src/metrics.rs

```diff
@@ -90,6 +103,59 @@ pub fn get_metrics(_: &protocols::agent::GetMetricsRequest) -> Result<String> {
     Ok(String::from_utf8(buffer)?)
 }
 
+#[instrument]
+fn register_metrics() {
+    REGISTRY
+        .register(Box::new(AGENT_SCRAPE_COUNT.clone()))
+        .expect(REGISTER_MSG);
```

Comment:
Please let's handle errors instead of panicing here.

Reply:
Fixed